packages/vue-use-tippy/source/index.ts

```diff
@@ -22,27 +23,33 @@ const applyForEvery = (
 			callback(tippyInstance)
 }
 
+const unwrapAndThrowOnNull = (target: Target) => {
+	if (target.value === null) throw new Error('useTippy: Unexpected null target')
+
+	return target.value
+}
+
 /**
  * @see {@link https://atomiks.github.io/tippyjs/v6/all-props} for options
  */
 export const useTippy = (
-	targets: Array<Target> | Target,
+	targets: Array<Target> | Target | string,
 	options: Ref<Partial<Props>>,
 ) => {
 	const instance = ref<InstanceRefType>(null)
 
 	const onMountCallbacks: Callback[] = []
 	onMounted(() => {
-		const unwrappedTargets = (() => {
-			if (isRef(targets)) return targets.value
-
-			if (Array.isArray(targets))
-				return targets.map((target) => (isRef(target) ? target.value : target))
-
-			return targets
-		})()
-
-		instance.value = tippy(unwrappedTargets, options.value)
+		if (typeof targets === 'string') {
```

Comment:
IIFE was removed in favor of a clear control flow for 1) readability
2) typescript compatability
because `tippy` has two signatures:
1. one that accepts `MultipleTargets`
2. one that accepts `SingleTarget`

using an IIFE to return unwrapped targets made both types unioned out of the outcome of that IIFE.. which confused typescript when we had one assignment statement
`instance.value = tippy(...)` because the signatures shouldn't overlap


Could have been fixed differently, for example, by calling `tippy` within the function expression BUT this way is more readable anyways